crates/core/src/db/datastore/system_tables.rs

```diff
@@ -800,6 +800,27 @@ impl From<&StClientsRow> for ProductValue {
     }
 }
 
+impl TryFrom<RowRef<'_>> for StClientsRow {
+    type Error = DBError;
+
+    fn try_from(row: RowRef<'_>) -> Result<Self, Self::Error> {
+        fn read_col<T: TryFrom<AlgebraicValue>>(row: RowRef<'_>, col: StClientsFields) -> Result<T, DBError> {
```

Comment:
Is there a less pedestrian way to do this? As in, can I not somehow `Deserialize` the whole `RowRef`?

/cc @Centril